dotcom-rendering/src/types/trails.ts

```diff
@@ -51,6 +51,7 @@ export interface CAPITrailType extends BaseTrailType {
 	pillar?: string;
 	carouselImages?: { [key: string]: string };
 	isLiveBlog?: boolean;
+	masterImage?: string;
```

Comment:
I fail to see why this is needed now, is there an error without it?

Reply:
If `masterImage` doesn't exist as a field in `CAPITrailType` there will be build failues. e.g. `make validate` fails with errors like this:
![image](https://user-images.githubusercontent.com/15894063/211033903-e3b9fb51-0bf6-448f-b595-0d3f8554d7a3.png)
Within the fixture files also, this error appears:
![image](https://user-images.githubusercontent.com/15894063/211034071-5244d66c-2655-4440-856a-a5c6dd4eed1a.png)